NUService.js

```diff
@@ -185,6 +185,17 @@ export default class NUService extends NUObject {
         });
     }
 
+    /*
+    *  Fetch health status.
+    *  component: name of the server component whose status is requested
+    */
+    fetchHealthStatus = (healthURL, component) => {
```

Comment:
There is no need to pass the healthURL it's set in stone just component

Reply:
the root URL is different for heath, it does not have `api/v5_0`. 
for example, it is

`https://10.31.100.64:8443/nuage` instead of `https://10.31.100.64/nuage/api/v5_0`

if I pass just the component, then I will have to compute root URL for health locally as,

``const rootURL = `${this.rootURL.substring(0, this.rootURL.indexOf('nuage') + 5)}`;``

Instead, I compute this URL in the caller using the org name constant, and pass healthURL as `https://10.31.100.64:8443/nuage/health`

Any other suggestion?